packages/app/src/components/About.tsx

```diff
@@ -0,0 +1,76 @@
+import styled from "styled-components";
+
+const StyleBox = styled.div`
+    display: flex;
+    flex-direction: column;
+    background-color: #48412f;
+    text-align: center;
+    padding: 2rem;
+    height: 40rem;
+`;
+
+const StyleSpan = styled.span`
+    margin-right: 10px;
+`;
+
+
+function About() {
+    return (
+        <StyleBox>
+            <h1>About</h1>
+            <p>
+            Welcome to Catchmaster, your ultimate fishing companion!
+            We understand that fishing isn't just a hobby; it's a passion, a way of life. 
+            Our app is designed to unite anglers from all walks of life, creating a vibrant community where you can share your fishing triumphs with fellow enthusiasts. 
+            Whether you've reeled in a trophy-sized bass, a colorful trout, or a feisty marlin, Catchmaster is your platform to showcase your prized catches. 
+            With a photo and a few words to capture the moment, you can connect with like-minded individuals, learn new techniques, and celebrate the thrill of the catch together. 
+            Join us at Catchmaster, just log in easily with your Google account and start sharing your fishing adventures today!
+            </p>
+            <p>Our group is composed of the following members:</p>
+            <ul>
+            <div>
+                    <StyleSpan>Adnan Avni</StyleSpan>
+                    <a href="https://www.instagram.com/adnanavni">
+                        <img width="20" height="20" src="https://img.icons8.com/ios/50/instagram-new--v1.png" alt="instagram-new--v1"/>
```

Comment:
```suggestion
                        <StyledAuthorImage src="https://img.icons8.com/ios/50/instagram-new--v1.png" alt="instagram-new--v1"/>
```
maybe

Reply:
Actually there is already the <Image> component that you guys have implemented before, that could possibly be re-used here.